### DISCUSSION TITLE
AUR for arch?

### DISCUSSION THREAD

**Discussion by sfxworks** [ltsp/ltsp#507] [General]
Any plans for one/am I missing one/should I attempt to make one?

**Comment by alkisg** [comment]
Hi @sfxworks, AFAIK there's no AUR package for LTSP, and personally I've no plans to create one at the moment (maybe in a couple of years!).

As with RPM, I welcome such attempts and I can provide advice and merge PRs upstream, but currently I don't have enough time to implement them myself.
